packages/react-components/src/components/SendBox.tsx

```diff
@@ -266,16 +274,23 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   const mergedStyles = useMemo(() => concatStyleSets(styles), [styles]);
 
   const hasText = !!textValue;
+  const hasFile = (): boolean => {
```

Comment:
I would extract this as a file private helper (trampoline) function rather than creating a new lambda on each render.